### DISCUSSION TITLE
Support .ico icons (image/x-icon)

### DISCUSSION THREAD

**Discussion by mathielen** [directus/directus#2740] [Feature Requests]
Hey again,

- The old directus 6.x did support icons and saved them as type="image/x-icon").
- After migrating the 6.x data to directus 8.7.2, the file-selection popup breaks if an icon would be shown
- I figured that if a user uploads an .ico file to directus 8.7.2. it is saved as type="application/ico", this will not break the file-selection popup anymore, but also no thumbnail is shown

So I guess .ico files are not supported anymore.
It would be nice if it would be supported again. We are dealing with favicon files for our sites, which may be .ico files. Would be cool to see the thumbnail of it.

Thank you

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
